clients/ui/docs/kubeflow-development-guide.md

```diff
@@ -0,0 +1,52 @@
+# Set up local development with kubeflow
+
+**Note: this should only be needed in edge cases in which we need to test a local integration with the kubeflow dashboard.**
+
+## Prerequisites
+
+- [Kubeflow repo](https://github.com/kubeflow/kubeflow/tree/master/components/centraldashboard#development)
+- [Model Registry repo](../README.md)
+
+## Setup
+
+### Kubeflow repo (under centraldashboard)
+
+1. Change the [webpack config](https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/webpack.config.js#L186) proxies to allow Model Registry:
+
+```javascript
+        proxy: {
+            ...
+            '/model-registry': {
+                target: 'http://localhost:9000',
+                changeOrigin: true,
+                ws: true,
+                secure: false,
+            },
+        },
+```
+
+2. Run the centraldashboard:
+
+```shell
+npm run dev
+```
+
+### Model Registry repo
+
+1. Just run the repo in kubeflow dev mode
+
+```shell
+make dev-start-kubeflow
```

Comment:
```suggestion
cd clients/ui && make dev-start-kubeflow
```

Reply:
I believe the commands in all the `clients/ui/docs` are relative to the `clients/ui` dir